src/modules/layout/components/LocationFooter.js

```diff
@@ -2,19 +2,22 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { translate } from 'react-i18next'
 
-import Navigation from 'modules/app/Navigation'
 import Footer from './Footer'
 import { Link } from 'redux-little-router'
+import DisclaimerPage from '../../../routes/disclaimer/containers/DisclaimerPage'
 
 class LocationFooter extends React.Component {
   static propTypes = {
-    navigation: PropTypes.instanceOf(Navigation).isRequired,
+    matchRoute: PropTypes.func.isRequired,
+    currentParams: PropTypes.object.isRequired,
     t: PropTypes.func.isRequired
   }
 
   render () {
     return <Footer>
-      <Link href={this.props.navigation.disclaimer}>{this.props.t('imprintAndContact')}</Link>
+      <Link href={this.props.matchRoute(DisclaimerPage).stringify(this.props.currentParams)}>
```

Comment:
why do you need the whole DisclaimerPage here to check for the route? wouln't 'DisclaimerPage' be sufficient?

Reply:
The ids are actual class references here. So a string wouln't be enough.